actions/checkout_remote_branch.yaml

```diff
@@ -0,0 +1,20 @@
+---
+name: checkout_remote_branch
+runner_type: remote-shell-cmd
+description: "checkout a remote branch for a repository"
+enabled: true
+entry_point: ''
+parameters:
+  local_repo_directory:
+    type: string
+    description: Local path to repo directory
+    required: true
+    position: 1
+  remote_branch:
+    type: string
+    description: remote branch to checkout
+    required: true
+    position: 2
+  cmd:
+    default: "cd {{ local_repo_directory }};git fetch;git checkout {{ remote_branch }}"
```

Comment:
I wonder if it's better do do something like this:
`cd {{ local_repo_directory }} && git fetch && git checkout {{ remote_branch }}`

That way if `local_repo_directory` doesn't exist, it will fail early, rather than checking out code to an unexpected location

Reply:
I think that's a really good call. Failing asap and preventing actions that cause unforeseeable changes is preferable in my book. I'll update.